pipelines/internal/commands/run/run.go

```diff
@@ -175,6 +175,26 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 		return fmt.Errorf("building request: %v", err)
 	}
 
+	if *pmbAttempts > 0 {
```

Comment:
I think you can replace all of this with

return runPipline(...)

and then in runPipeline just do something like:

for {
  req.Pipeline.Resources.VirtualMachine.Preemptible = (attempt < *pvmAttempts)
  ...
  if attempt < *pvmAttempts + *attempts {
  }
}

right?